app/code/community/Razorpay/Payments/Helper/Data.php

```diff
@@ -100,11 +102,20 @@ protected function getRazorpayOrderData($order)
         return $data;
     }
 
+    protected function getOrderAmountInInr($orderAmount, $orderCurrency)
+    {
+        $url = "http://api.fixer.io/latest?base=$orderCurrency";
+
+        $rates = json_decode(file_get_contents($url), true);
+
+        return $orderAmount * $rates['rates']['INR'];
```

Comment:
@mayankamencherla this will return a float.

Reply:
@ccrims0n Using 
```
return (int) round($orderAmount * $rates['rates']['INR'], 2);
```